Ical.Net/DataTypes/CalDateTime.cs

```diff
@@ -261,25 +255,33 @@ public override int GetHashCode()
     }
 
     public static bool operator <(CalDateTime? left, IDateTime? right)
-        => left != null
-           && right != null
-           && (left.IsFloating || right.IsFloating ? left.Value < right.Value : left.AsUtc < right.AsUtc);
+    {
+        return left != null
+               && right != null
+               && (left.IsFloating || right.IsFloating ? left.Value < right.Value : left.AsUtc < right.AsUtc);
```

Comment:
Here additional parenthesis could aid readability `((left.IsFloating || right.IsFloating) ? left.Value < right.Value : left.AsUtc < right.AsUtc)`, as the precedence between `||` and the ternary operator might not be immediately obvious to everyone.

Reply:
makes sense indeed, tnx